src/main/kotlin/no/nav/hm/grunndata/register/product/ProductRegistrationApiController.kt

```diff
@@ -211,6 +211,21 @@ class ProductRegistrationApiController(
         return HttpResponse.ok(updated)
     }
 
+    @Delete("/draft/delete")
+    suspend fun deleteDraftVariants(
+        @Body ids: List<UUID>,
+        authentication: Authentication,
+    ): HttpResponse<List<ProductRegistrationDTO>> {
+        val products = productRegistrationService.findByIdIn(ids).onEach {
+            if (it.supplierId != authentication.supplierId()) return HttpResponse.unauthorized()
+            if (!(it.draftStatus == DraftStatus.DRAFT && it.published == null)) throw BadRequestException("product is not draft")
+        }
+
```

Comment:
jeg ville lagt inn delete called for id $id and supplierRef, i tilfellet noen lurte hvor ting er blitt av